client/src/components/AuthRoute/AuthRoute.tsx

```diff
@@ -0,0 +1,7 @@
+import { Redirect, Route, RouteProps } from 'react-router-dom';
+import { useAuth } from '../../context/useAuthContext';
+
+export const AuthRoute = (props: RouteProps) => {
```

Comment:
Optional: Maybe we could add a loading spinner for the split second when the user is directed to the login page before being redirected to the dashboard (when we refresh the page)